dietpi/dietpi-software

```diff
@@ -13213,8 +13213,14 @@ _EOF_
 		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == -1 )); then
 
 			Banner_Uninstalling
+			# Stop  Lighttpd service
+			[[ -f '/etc/systemd/system/lighttpd.service' ]] && systemctl disable --now lighttpd
+
 			G_AGP lighttpd
 
+			# Remove enabled Lighttpd configuration directory
+			[[ -d '/etc/lighttpd/conf-enabled' ]] && rm -R /etc/lighttpd/conf-enabled
```

Comment:
Interesting. Probably we should remove the whole `/etc/lighttpd` then. E.g. on Pi-hole uninstall `/etc/lighttpd/conf-available` is checked, so that should be avoided then as well.

Reply:
Was thinking the same and this was my first try but changed it to the sub directory to keep user created configuration files if available. But we could remove the entire folder. Just need to agree on 😉

Reply:
At least removing the whole dir is how we do it in other cases as well. Uninstalling software basically means purging data and config, which is also stated before confirming the uninstall. A differentiation between "uninstall"/"remove" and "purge", like APT has it, would be another nice feature for the future, but until then, I prefer to remove things cleanly 🙂.

Reply:
Ok I will change to have the entire folder removed